src/Providers/LaradiumServiceProvider.php

```diff
@@ -283,8 +291,25 @@ private function publishAssets()
      */
     private function registerViewComposers()
     {
-        View::composer(
-            'laradium::layouts.main', 'Laradium\Laradium\ViewComposers\VariableComposer'
-        );
+        $composers = [
```

Comment:
Is it really necessary to create an array and loop through it? 

Why not just call the view()->composer() method 3 times, instead of wasting memory and CPU cycles?